snapshots/education/2018-04-18/literacy_rates.feather.dvc

```diff
@@ -1,26 +1,78 @@
 meta:
   source:
     name: Literacy rates (World Bank, CIA World Factbook, and other sources)
-    description: |-
+    description: >-
       Additional Information
 
-      This long run cross-country dataset combines data from a number of sources. We took the estimates from the World Bank’s WDI as our base, and then extended coverage by adding literacy estimates from the CIA Factbook, as well as several other long-run series, as follows:
+      </br>
 
-      - Data before 1800: Buringh, E., & Van Zanden, J. L. (2009). Charting the “Rise of the West”: Manuscripts and Printed Books in Europe, A long-term perspective from the sixth through eighteenth centuries. The Journal of Economic History. Online [here](http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.553.9220&rep=rep1&type=pdf). Observations before 1800 are plotted at the midpoint of the given time range (1475 refers to 1451–1500, 1550 refers to 1501-1600 etc.)
-      - Data for 1820 and 1870 (except for the US): Broadberry and O'Rourke (2010) – The Cambridge Economic History of Modern Europe: Volume 1, 1700-1870
-      - Data for the US: National Center for Education Statistics, online [here](nces.ed.gov/naal/lit_history.asp).
-      - Global estimates for 1820-2000: van Zanden, J.L., et al. (eds.) (2014), How Was Life?: Global Well-being since 1820, OECD Publishing. Online [here](http://www.oecd.org/statistics/how-was-life-9789264214262-en.htm).
-      - Historical estimates for Latin America: OxLAD – Oxford Latin American Economic History Data Base, online [here](http://moxlad-staging.herokuapp.com/home/en).
-      - Most recent estimates for high-income countries, as well as any available estimates for 2016: [CIA World Factbook](https://www.cia.gov/library/publications/the-world-factbook/).
+      This long run cross-country dataset combines data from a number of sources.
```

Comment:
I think we've recently switched to Markdown for Sources. HTML still gets rendered, but we should write all new metadata in Markdown (I'm like 95% sure...)

Reply:
ah ok I see! I've added some code to do the conversion from html to markdown in the snapshot step and I think it looks ok!

Reply:
Oh, nice one! I might add it to the backporting template.

Reply:
sounds good! I am happy to merge whenever now. This will trigger a rebuild of a pretty huge WB education dataset so I usually do it at the end of the day :) but not sure if this is actually something I needed to worry about

Reply:
Merge at will!